CatAnalyzer/test/run_TtbarDiLeptonAnalyzer_cfg.py

```diff
@@ -104,7 +105,8 @@
     partonTop_modes = cms.InputTag("partonTop", "modes"),
     partonTop_genParticles = cms.InputTag("partonTop"),
 
-    isTTbarMC = cms.bool(False),
+    #isTTbarMC = cms.bool(False),
+    isTTbarMC = cms.bool(runOnMC),
```

Comment:
most recent cattuple for background MC's are produced without partontop/pseudotop in the event content. It may crash without isTTbarMC flag.